flags/conditions.py

```diff
@@ -96,7 +94,8 @@ def parameter_condition(param_name, request=None, **kwargs):
         raise RequiredForCondition("request is required for condition "
                                    "'parameter'")
 
-    return request.GET.get(param_name) == 'True'
+    return param_name in request.GET
```

Comment:
What happens if you try to access a URL with an empty query string parameter? https://example.com/?foo=

Reply:
The flag should evaluate to `True`. This seems sensible since there's no way to specify a particular expected value — when you create the condition, you're simply giving it a parameter name to expect. Therefore it seems sensible to make that any value (including empty) should match.

Reply:
Thanks, that makes sense to me.